docs/user/index.rst

```diff
@@ -0,0 +1,40 @@
+User Guide
+==========
+
+This section of the documentation provides information about how to install
+uvloop as well as how to use uvloop.
```

Comment:
I'd rephrase to:

    This section of the documentation provides information about 
    how to use uvloop.